### ISSUE TITLE
[AdaptiveCard styling] Rendered AdaptiveCards missing Buttons in Dark Theme

### ISSUE THREAD

**Issue by mathewleeza** [microsoft/devhomegithubextension#107]
### Steps to reproduce the bug

Light theme:
![image](https://github.com/microsoft/devhome/assets/111322232/aa51595c-8d16-4e58-acca-16e8cf46e727)

Dark Theme:
Image
![image](https://github.com/microsoft/devhome/assets/111322232/76b40f6e-9d3e-4e93-a20a-e0f60e3c2905)

Button could just be behind something else, but this is happening only for DarkTheme. Button is not clickable either (so not just invisible).

**Comment by krschau** [comment]
Fixed by https://github.com/microsoft/devhome/pull/304
